src/viam/ur/module/ur_arm.cpp

```diff
@@ -111,22 +111,24 @@ void write_joint_data(const vector6d_t& jp, const vector6d_t& jv, std::ostream&
 }
 
 std::vector<std::string> validate_config_(const ResourceConfig& cfg) {
-    static_cast<void>(find_config_attribute<std::string>(cfg, "host"));
-    static_cast<void>(find_config_attribute<double>(cfg, "speed_degs_per_sec"));
-    static_cast<void>(find_config_attribute<double>(cfg, "acceleration_degs_per_sec2"));
-    {
-        auto threshold = -1.0;
-        try {
-            threshold = find_config_attribute<double>(cfg, "reject_move_request_threshold_deg");
-        } catch (...) {
-            // will go away once find_config_attribute returns an optional
-            return {};
-        }
-        if (threshold < 0 || threshold > 360) {
-            std::stringstream sstream;
-            sstream << "reject_move_request_threshold_deg should be between 0 and 360, it is : " << threshold;
-            throw std::invalid_argument(sstream.str());
-        }
+    if (!find_config_attribute<std::string>(cfg, "host")) {
+        throw std::invalid_argument("attribute `host` is required");
+    }
+    if (!find_config_attribute<double>(cfg, "speed_degs_per_sec")) {
+        throw std::invalid_argument("attribute `speed_degs_per_sec` is required");
+    }
+    if (!find_config_attribute<double>(cfg, "acceleration_degs_per_sec2")) {
+        throw std::invalid_argument("attribute `acceleration_degs_per_sec2` is required");
+    }
+
+    auto threshold = find_config_attribute<double>(cfg, "reject_move_request_threshold_deg");
+    constexpr double MIN_THRESHOLD = 0.0;
+    constexpr double MAX_THRESHOLD = 360.0;
```

Comment:
I like these as `k_min_threshold` and `k_max_threshold`. C++ doesn't mandate all caps for constants (I think rust got this wrong), and I like this article for justification: https://accu.org/journals/overload/22/121/wakely_1923/